src/assets/class-manifest-helper.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * The Manifest Helper specific functionality.
+ * USE ONLY IN VIEWS WHERE YOU CANT CONTROL RENDERING METHODS.
+ *
+ * @since   4.0.0 Init class.
+ * @package Inf_Theme\Assets
+ */
+
+namespace Inf_Theme\Assets;
+
+use Inf_Theme\Assets\Manifest;
+
+/**
+ * Class Manifest_Helper
+ */
+class Manifest_Helper {
+
+  /**
+   * Set Assets Manifest global variable name.
```

Comment:
Method name and the fact that it has `return` means that this is not a setter but a getter 🙂

Reply:
👍